react@"^17.0.2

please provide support for react 17 and 18. It actually runs nicely with 17. 

`
$ npm i react-overlay-component --save
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: project@3.0.0
npm ERR! Found: react@17.0.2
npm ERR! node_modules/react
npm ERR!   react@"^17.0.2" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^15.0.0 || ^16.0.0" from react-overlay-component@1.0.2
npm ERR! node_modules/react-overlay-component
npm ERR!   react-overlay-component@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
`

--- a/package.json
+++ b/package.json
@@ -32,8 +32,8 @@
     },
     "peerDependencies": {
         "prop-types": "^15.5.4",
-        "react": "^15.0.0 || ^16.0.0",
-        "react-dom": "^15.0.0 || ^16.0.0"
+        "react": "^15.0.0 || ^16.0.0 || ^17.0.0",
+        "react-dom": "^15.0.0 || ^16.0.0 || ^17.0.0"
     },
     "devDependencies": {
         "@babel/cli": "^7.5.5",

